component/core/pom.xml

```
@@ -28,7 +28,7 @@
   <name>Meeds:: PLF:: Social Core Component</name>
   <description>Meeds Social Core Component: People and Space</description>
   <properties>
-    <exo.test.coverage.ratio>0.72</exo.test.coverage.ratio>
+    <exo.test.coverage.ratio>0.71</exo.test.coverage.ratio>
```

We may need to increase it

```suggestion
    <exo.test.coverage.ratio>0.72</exo.test.coverage.ratio>
```